src/helpers.js

```diff
@@ -313,7 +333,13 @@ function processNode(node, formValues, formFields, accRequired = new Set()) {
       const inputType = getInputType(nestedNode);
       if (inputType === supportedTypes.FIELDSET) {
         // It's a fieldset, which might contain scoped conditions
-        processNode(nestedNode, formValues[name] || {}, getField(name, formFields).fields);
+        processNode({
+          node: nestedNode,
+          formValues: formValues[name] || {},
+          formFields: getField(name, formFields).fields,
+          validations,
```

Comment:
Tiny nitpick - in every other instance above we're passing the object properties in the same order except here. Can this one be updated to be `parentID`, `validations`? It just makes it easier to read - I did a double take to see if anything was missing here since it wasn't the same.

Reply:
Thats fair, changing